<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>701</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Post</string>
   </property>
   <widget class="QLabel" name="label_url">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Url</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_url">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Up</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Down</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_type">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Type :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Count Of Upvote or Downvote :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_number_account">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of accounts/windows  : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_vote">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>141</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_account">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>551</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Comment</string>
   </property>
   <widget class="QLabel" name="label_url_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Url</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_url_comment">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type_comment">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Up</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Down</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_type_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Type :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count1_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Count Of Upvote or Downvote :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_number_account_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of accounts/windows  : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_vote_comment">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>141</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_account_comment">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start_comment">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
